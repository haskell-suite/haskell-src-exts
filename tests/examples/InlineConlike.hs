{-# INLINE CONLIKE size #-}
size :: Int
size = 64