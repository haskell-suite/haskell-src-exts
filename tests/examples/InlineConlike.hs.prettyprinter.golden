module Main (main) where
 
{-# INLINE_CONLIKE size #-}
 
size :: Int
size = 64